src/algebra/opposites.lean

```diff
@@ -35,6 +35,26 @@ instance [add_comm_semigroup α] : add_comm_semigroup (opposite α) :=
 instance [has_zero α] : has_zero (opposite α) :=
 { zero := op 0 }
 
+@[simp] lemma zero_unop [has_zero α] : (0 : αᵒᵖ).unop = (0 : α) := rfl
+@[simp] lemma zero_op [has_zero α] : op (0 : α) = (0 : αᵒᵖ) := rfl
+
+@[simp] lemma unop_eq_zero_iff [has_zero α] (a : αᵒᵖ) : a.unop = 0 ↔ a = 0 :=
```

Comment:
Is it an `iff.rfl` with a `reducible` `unop` and `op`?

Reply:
You can also try `by rw [eq_comm, ← op_eq_iff_eq_unop, zero_op]` (possibly with one more `eq_comm`).